### PR TITLE
Adds a new disease: Carcinization

### PR DESCRIPTION
**Pull request by neeshacark**
# Document the changes in your pull request

image
image

adds a disease which turns you into a crab

# Why is this good for the game?
idk man ask cuackles

# Testing
trust

# Changelog

:cl:  cark
rscadd: Added a new disease which turns you into a crab
/:cl:

### CODE REVIEW

**Comment by Yarinoi** [comment]
the ultimate lifeform

File: code/datums/diseases/transformation.dm [hunk 1]
```diff
@@ -287,6 +287,22 @@
...
+	cures = list(/datum/reagent/cellulose) // opposite of crab = barc = bark = cellulose
+	agent = "Carcinization"
+	desc = "Nature's next attempt to evolve you into a crab."
+	stage_prob = 20
+	severity = DISEASE_SEVERITY_BIOHAZARD
+	visibility_flags = 0
+	stage1	= list("Your hands itch.")
+	stage2	= list("Your skin feels crusty.")
+	stage3	= list(span_danger("Your hands are forming into claws."), span_danger("Your limbs begin to become crablike."))
+	stage4	= list(span_danger("You're raveing."))
```
Comment (AMyriad) [inline code/datums/diseases/transformation.dm:302]: this would be spelled without the "e" 👀

**Comment by MajManatee** [comment]
A virus that makes you into Cark...
worse than death

And how is it contracted?

File: code/datums/diseases/transformation.dm (hunk 1 above)
Comment (neeshacark) [reply code/datums/diseases/transformation.dm:302]: No its spelt raveing intentionally

as in rave

as in https://www.youtube.com/watch?v=LDU_Txk06tM

**Comment by neeshacark** [comment]
> A virus that makes you into Cark... worse than death
> 
> And how is it contracted?

its currently only spawnable via admins, i only did this cause cuackles asked

**Comment by azzzertyy** [comment]
> A virus that makes you into Cark... worse than death
> 
> And how is it contracted?

manatee clearly does not read code

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**